server/src/assets/entities/asset.entity.ts

```diff
@@ -32,5 +33,10 @@ export class Asset {
   quantity: number;
 
   @ManyToOne(() => User, (user) => user.assets)
+  @JoinColumn()
```

Comment:
👍 
good catch, can you do a quick audit of the rest of our entities and see if there are any more of these we missed? put anything in a separate PR
if you already had plans to do something else lmk and i'll throw in an issue

Reply:
https://typeorm.io/#/undefined/creating-a-one-to-one-relation
(for resources list in pr/issue)

Reply:
Currently none of the entities in the app use @JoinColumn().    I'm putting this here so I remember it:    https://orkhan.gitbook.io/typeorm/docs/relations   I think I need to digest what's meant by a Join Column.   It looks like what they're saying is that you only want it in a table that has a foreign key, is that correct?

Reply:
yes, i think it's an explicit way to set the foreign key. otherwise it's left up to nest to figure it out (which is probably fine for our use cases, but doesn't hurt to have it)

Reply:
https://github.com/Nonprofit-Exchange-Hub/web-app/issues/127